### ISSUE TITLE
7. Ontbrekende Main Landmark  (API's template A11y)

### ISSUE THREAD

**Issue by TessaViergever** [frameless/developer.overheid.nl#107]
### Betrokken pagina's

[Zoek API's](https://rijkshuisstijl-community-templates-3losjges9-nl-design-system.vercel.app/developer-overheid/api-overzicht)

### Beschrijf het probleem

De webpagina mist een Main Landmark.

### Relevante code

```html
`<body class="rhc-theme">`
```

### WCAG-richtlijn

WCAG Best Practice

### Voeg een afbeelding toe (indien relevant)

<img width="783" alt="A widget shows the different landmarks present on the page. The page lacks a Main landmark." src="https://github.com/user-attachments/assets/005e31aa-752e-402f-b889-0f71c14588d5" />

### Referentienummer

TBD
